### DISCUSSION TITLE
latest_testflight_build_number

### DISCUSSION THREAD

**Discussion by ACommonChinese** [fastlane/fastlane#19127] [General] [answered]
Today, when i run latest_testflight_build_number, it shows:
[!] The request could not be completed because:
	Not Found

**Comment by crazymanish** [comment, marked as answer]
Hi @ACommonChinese 

Please follow the update here:
- https://github.com/fastlane/fastlane/issues/19124
- https://github.com/fastlane/fastlane/pull/19131

**Reply by crazymanish** [reply]
This has been fixed in latest release `2.188.0` https://github.com/fastlane/fastlane/releases/tag/2.188.0
